stable_baselines/common/vec_env/dummy_vec_env.py

```diff
@@ -81,3 +81,15 @@ def _obs_from_buf(self):
             return self.buf_obs[None]
         else:
             return self.buf_obs
+
+    def env_method(self, method_name, *method_args, **method_kwargs):
+        """
+        Provides an interface to call arbitrary class methods of vectorized environments
+
+        :param method_name (str) The name of the env class method to invoke
+        :param method_args (tuple) Any positional arguments to provide in the call
+        :param method_kwargs (dict) Any keyword arguments to provide in the call
+        :return: (list) List of items retured by the environment's method call
+        """
+        method = getattr(self.envs[0], method_name)
```

Comment:
This won't work for n_envs > 1, you should do a for loop to retrieve all the items returned.

Reply:
ok.  just glanced at a function above which was dealing with self.envs[0] and thought the dummies were limited to len(dummy)==1.

Reply:
DummyVecEnv is usually only used with n_env=1, however, it is also useful for debugging (for comparisons with subprocvecenv) to have n_env > 1